dts/bindings/mbox/ti,secure-proxy.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2025 Texas Instruments Incorporated.
+# SPDX-License-Identifier: Apache-2.0
+
+description: TI AMX Secure Proxy MAILBOX
```

Comment:
It isn't just AMx devices, this IP is found in J7x and other TI SoCs. Suggest:

`description: TI Secure Proxy MAILBOX`